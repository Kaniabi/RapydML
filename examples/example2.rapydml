div(id="content"):
    "I haven't yet put anything on this page"
    p:
        "Compatible with all major browsers:"
        for $browser in [Firefox, Chrome, IE, Opera, Safari]:
            img(src="$browser.jpg", alt="$browser")
