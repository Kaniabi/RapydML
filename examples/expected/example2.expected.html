<div id="content">
    I haven't yet put anything on this page
    <p>
        Compatible with all major browsers:
        <img src="Firefox.jpg" alt="Firefox" />
        <img src="Chrome.jpg" alt="Chrome" />
        <img src="IE.jpg" alt="IE" />
        <img src="Opera.jpg" alt="Opera" />
        <img src="Safari.jpg" alt="Safari" />
    </p>
</div>
